sdk/dapp-kit/test/components/SuiClientProvider.test.tsx

```diff
@@ -101,4 +101,109 @@ describe('SuiClientProvider', () => {
 
 		expect(screen.getByText('selected network: b')).toBeInTheDocument();
 	});
+
+	test('controlled mode', async () => {
+		function NetworkSelector(props: { selectNetwork: (network: string) => void }) {
+			const ctx = useSuiClientContext();
+
+			return (
+				<div>
+					<div>{`selected network: ${ctx.network}`}</div>
+					{Object.keys(ctx.networks).map((network) => (
+						<button key={network} onClick={() => props.selectNetwork(network)}>
+							{`select ${network}`}
```

Comment:
this is just a super nitty thing but it's a little more concise/standard to write strings like this without using template literals

```
<div>selected network: {ctx.network}</div>
```

Reply:
this was causing some issues with selecting by text, but I didn't dig too deeply into what was going on...